spec/03-plugins/15-response-transformer/02-body_transformer_spec.lua

```diff
@@ -135,6 +135,11 @@ describe("Plugin: response-transformer", function()
         assert.same({a = {}}, body_json)
         assert.equals('[]', cjson.encode(body_json.a))
       end)
+      it("plain text", function()
+        local body = "test"
+        local json_body = body_transformer.transform_json_body(conf, body)
+        assert(json_body==nil)
```

Comment:
style: `json_body == nil`

Reply:
@samugi Yes, a warning would be useful. Thanks!
@chronolaw Style has been corrected. Thanks!